packages/fast-components-styles-msft/src/design-system/index.ts

```diff
@@ -209,13 +209,24 @@ export const withDesignSystemDefaults: (config: Partial<DesignSystem>) => Design
 
 /**
  * Safely retrieves a single property from a design system
+ * @deprecated Use getDesignSystemValue instead.
  */
 export function getDesignSystemProperty(key: string): DesignSystemResolver<string> {
     return function(config: DesignSystem): string {
         return withDesignSystemDefaults(config)[key];
     };
 }
 
+/**

```

Comment:
I actually think these should live in `utilities/design-system` - they are utility functions after all